sentry-samples/sentry-samples-android/src/main/AndroidManifest.xml

```diff
@@ -72,6 +74,15 @@
         <!--    how to change the session tracking interval-->
         <meta-data android:name="io.sentry.session-tracking.timeout-interval-millis" android:value="10000" />
 
+        <!--    how to enable the performance API by setting a sample-rate-->
+        <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
+
+        <!--    how to disable the Activity auto instrumentation for tracing-->
+        <!--    <meta-data android:name="io.sentry.traces.activity.enable" android:value="false" />-->
+
+        <!--    how to disable the Activity auto instrumentation for tracing (finishing automatically)-->
+        <!--    <meta-data android:name="io.sentry.traces.activity.finish.enable" android:value="false" />-->
```

Comment:
For clarity here I suggest a change in the name of the options:

```suggestion
        <!--    how to disable the Activity auto instrumentation automatically finished on onClose -->
        <!--    <meta-data android:name="io.sentry.traces.activity.auto-finish.enable" android:value="false" />-->
```

Reply:
yeah, makes sense, renamed the options too to `enableActivityLifecycleTracingAutoFinish`